packages/facade/src/apis/sheets/f-range.ts

```diff
@@ -255,6 +255,24 @@ export class FRange {
         return range;
     }
 
+    /**
+     * Returns the cell data for the cells in the range.
+     * @returns A two-dimensional array of cell data.
+     */
+    getCellDatas(): Nullable<ICellData>[][] {
```

Comment:
`Datas` is not a canonical plural form, consider `getCellDataGrid` or `getCellDataTable`.